srcpkgs/webhook/files/webhook/conf

```diff
@@ -0,0 +1,122 @@
+# A list of paths to json files containing defined hooks webhook should serve.
+# Each filepath should be separated by whitespace, and quoted if needed.
+#
+# Default: /etc/webhook/hooks.json
+#
+# HOOKS_FILES="/etc/webhook/hooks.json"
```

Comment:
I dislike the idea of vendoring in this file. Does upstream not provide an example config? :/

This file doesn't seem to be installed at all? Maybe it could be omitted.

Reply:
there is an example config. I could use that, but it has stuff like hard-coded paths to the dev's $HOME so it feels a bit weird: https://github.com/adnanh/webhook/blob/master/hooks.json.example

Reply:
per the advice of upstream (adnanh/webhook#560), I've updated the template to include a default `/etc/webhook/hooks.json` that simply runs `date`.

Reply:
Maybe replace the `"response-message": "I got the payload!"` with `"include-command-output-in-response": true`?